packages/alert/src/types.ts

```diff
@@ -23,10 +41,6 @@ type AlertProps = {
    * Title of the alert (applies only if style is not simple)
    */
   title?: string
-  /**
-   * Hide the action button (applies only if style is not simple)
-   */
-  hideAction?: boolean
```

Comment:
I need this property for the alert inside Dichiarazioni (it's a collapsable alert without an explicit action or link)